Generals/Code/GameEngineDevice/Source/W3DDevice/GameClient/W3DShaderManager.cpp

```diff
@@ -2844,7 +2843,10 @@ Bool W3DShaderManager::testMinimumRequirements(ChipsetType *videoChipType, CpuTy
 
 	if (intBenchIndex && floatBenchIndex && memBenchIndex)
 	{
-		RunBenchmark(0, NULL, floatBenchIndex, intBenchIndex, memBenchIndex);
+        // TheSuperHackers @Tweak Aliendroid1 19/June/2025 Tests redundant for modern hardware so we assume passing values. 
```

Comment:
`@` keywords are always lowercase.

Date just contains numbers, no words: 19/06/2025.

Comment can be worded better, such as "Now always returns good values because the legacy benchmark code was removed"